deployment.md

```diff
@@ -0,0 +1,257 @@
+# LinkedTrust Deployment and Maintenance Guide
+
+This guide provides detailed instructions for deploying and maintaining the LinkedTrust project on a production server.
+NB: Statics is served using whitenoise
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Initial Server Setup](#initial-server-setup)
+- [Project Deployment](#project-deployment)
+- [Database Setup](#database-setup)
+- [Static Files](#static-files)
+- [Process Management](#process-management)
+- [Nginx Configuration](#nginx-configuration)
+- [SSL/TLS Setup](#ssltls-setup)
+- [Maintenance](#maintenance)
+- [Troubleshooting](#troubleshooting)
+- [Backup Procedures](#backup-procedures)
```

Comment:
most important is which server is it on, and where is the vault key

like EXACTLY what does someone have to do to connect to it, if they have no context or knowledge of where it is @vortex-hue